CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * Add `IgnoredMetadata` configuration option to `RSpec/DescribeClass`. ([@Rafix02][])
 * Fix false positives in `RSpec/EmptyExampleGroup`. ([@pirj][])
 * Fix a false positive for `RSpec/EmptyExampleGroup` when example is defined in an `if` branch. ([@koic][])
+* Add RSpec/PersistenceCalledOutsideExample cop.([@thijsnado][])
```

Comment:
```suggestion
* Add RSpec/Rails/PersistenceCalledOutsideExample cop.([@thijsnado][])
```
Please also change PR title 😉

Reply:
👍 

For the reference: https://github.com/rubocop-hq/rubocop-rspec/pull/1019